keras_autodoc/autogen.py

```diff
@@ -67,9 +68,18 @@ def generate(self, dest_dir):
         if self.examples_dir is not None:
             copy_examples(self.examples_dir, dest_dir / "examples")
 
-    def process_docstring(self, docstring):
+    def process_docstring(self, docstring, types):
         """Can be overridden."""
-        return process_docstring(docstring)
+        processsed = process_docstring(docstring)
+
+        # Add type annotations to docstring
+        for arg, arg_type in types.items():
+            arg_type = getattr(arg_type, "__name__", arg_type)
+            processsed = re.sub(
+                rf"(- __{arg}__)", rf"\1 `{arg_type}`", processsed
+            )
```

Comment:
I think this piece of code would be better fitted at this line: https://github.com/keras-team/keras-autodoc/blob/0af2f382813f047a30faadfa9dd91f7f2c070e3b/keras_autodoc/docstring.py#L80  , with a check "if the section name is `Arguments`".

Reply:
Moved in 61654b8.